src/nxdoc/nuxeo-server/nuxeo-server-release-notes.md

```diff
@@ -544,6 +1115,13 @@ Ex (deployment-fragment.xml):
 
 <i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-25040](https://jira.nuxeo.com/browse/NXP-25040).
 
+#### Tomcat 9.0.14 {{since '10.10'}}
+
+The LTS 2019 version of Nuxeo is aligned on Tomcat 9.0.14
+
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-26573](https://jira.nuxeo.com/browse/NXP-26573).
+
+
```

Comment:
Might also want to mention that session cookie tracking can be configured and is enabled by default: https://jira.nuxeo.com/browse/NXP-16398.